packages/components/dialog/index.ts

```diff
@@ -6,3 +6,4 @@ export default ElDialog
 
 export * from './src/use-dialog'
 export * from './src/dialog'
+export * from './src/dialog-content'
```

Comment:
Then why you export dialog-content?

Reply:
This is used by `drawer` because some properties are the same, I do not want to create duplication so I simply exported it.

Reply:
So I think why don't you export DialogContent component, make it public.

Reply:
I will just stop contributing on this